lib/src/commons/BrowserUtils.js

```diff
@@ -508,8 +508,47 @@ var BrowserUtils;
     }
     BrowserUtils.expectAlertText = expectAlertText;
     /**
-     * When switching between iframes, without wait it will fail to switch to iframe
      *
+     * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+     */
+    function mouseButtonDown(mouseButton) {
+        //Defaults to the left mouse button if not specified.
+        const selectedMouseButton = mouseButton === undefined ? 'LEFT' : mouseButton;
```

Comment:
I prefer to force a choice instead of having a default value. It will make reading usage impossible without opening the source code of this method (`mouseButtonDown()` vs. `mouseButtonDown(MouseButton.LEFT)`)

Reply:
fixed to mandatory